src/components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components'
+import Container from '../layout/Container'
+
+const Outer = styled.div<{ background?: string }>`
+  background: ${({ theme, background }) => (!background ? theme.colors.gradients.bubblegum : background)};
```

Comment:
Just a detail, but when you can, try to not reverse a check, it's easier to read when there is no negation. i.e 
`(background ? background : theme.colors.gradients.bubblegum)`, that can probably be shorten into `(background || theme.colors.gradients.bubblegum)`

Reply:
Ah that's muchhh better, ty Doge.